src/ui/page/dataverse/dataspace/governance/governance.tsx

```diff
@@ -0,0 +1,90 @@
+import type { FC } from 'react'
+import { useState, useCallback, useEffect } from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { pipe } from 'fp-ts/lib/function'
+import type { Option } from 'fp-ts/Option'
+import * as O from 'fp-ts/Option'
+import type { DataSpace } from '@/ui/page/dataverse/dataverse'
+import { getResourceDetails } from '@/ui/page/dataverse/dataverse'
+import { isDataSpace } from '@/ui/page/dataverse/dataspace/dataspace'
+import { GovernanceContent } from './governanceContent'
+import { mockedGovernanceChapter } from './mockedData'
+import './governance.scss'
+
+// eslint-disable-next-line max-lines-per-function
+export const Governance: FC = () => {
+  const navigate = useNavigate()
+  const {
+    id: dataspaceId,
+    sectionId: sectionIdParams,
+    subsectionId: subsectionIdParams
+  } = useParams<string>()
+
+  const [dataspace, setDataspace] = useState<Option<DataSpace>>(O.none)
```

Comment:
Why do you need a state? 
Here you only want to compute an element from another and extract a value, there is no need to memoize the result in a local state, you should use a `useMemo` instead.